tests/functional/disaster-recovery/regional-dr/test_ols_recipe_generation.py

```
@@ -0,0 +1,925 @@
+    """
+
+    params = [
+        pytest.param(
+            constants.CEPHBLOCKPOOL,
+            marks=[pytest.mark.polarion_id("OCS-8231")],
+            id="rbd",
+        ),
+    ]
+
+    @pytest.fixture(scope="class")
+    def ols(self, skip_if_ols_not_available, request):
```

Ruff (0.16.3)

[warning] Unused method argument: `skip_if_ols_not_available`

(ARG002)